jsonschema-core.xml

```diff
@@ -1447,103 +1464,59 @@
+                    <section title='Dynamic References with "$dynamicRef"' anchor="dynamic-ref">
+                        <t>
+                            The "$dynamicRef" keyword is an applicator that allows for deferring the
+                            full resolution until runtime, at which point it is resolved each time it is
+                            encountered while evaluating an instance.
+                        </t>
+                        <t>
+                            Together with "$dynamicAnchor", "$dynamicRef" implements a cooperative
+                            extension mechanism that is primarily useful with recursive schemas
+                            (schemas that reference themselves).  Both the extension point and the
+                            runtime-determined extension target are defined with "$dynamicAnchor",
+                            and only exhibit runtime dynamic behavior when referenced with
+                            "$dynamicRef".
+                        </t>
                         <t>
-                            The "$dyanmicRef" and "$dyanmicAnchor" keywords are used to construct
-                            extensible recursive schemas.  A recursive schema is one that has
-                            a reference to its own root, identified by the empty fragment
-                            URI reference ("#").
+                            The value of the "$dynamicRef" property MUST be a string which is
+                            a URI-Reference.  Resolved against the current URI base, it produces
+                            the URI used as the starting point for runtime resolution.  This initial
+                            resolution is safe to perform on schema load.
                         </t>
                         <t>
-                            Simply stated, a "$dynamicRef" behaves identically to "$ref", except
-                            when its target schema contains "$dynamicAnchor" with a value of true.
-                            In that case, the dynamic scope is examined to determine a new base URI,
-                            and the URI-reference in "$dynamicRef" is re-evaluated against that
-                            base URI.  Unlike base URI changes with "$id", changes with
-                            "$dynamicAnchor" are calculated each time a "$dynamicRef" is
-                            resolved, and do not impact any other keywords.
+                            If the initially resolved starting point URI includes a fragment that
+                            was created by the "$dynamicAnchor" keyword, the initial URI MUST be
+                            replaced by the URI (including the fragment) for the outermost schema
+                            resource in the <xref target="scopes">dynamic scope</xref> that defines
+                            an identically named fragment with "$dynamicAnchor".
```

Comment:
I'm going to want an example (for me) of what this means. It may be it's simplt too late in the day for me to parse this paragraph, but actually that might not be a bad measure.

Reply:
@Relequestual does the example in the appendix not help?  I would like to work on that example until it is clear rather than try to come up with a different one.

Reply:
After taking another read, it makes sense. You just have to read it in the context of the previous paragraph. It may well have been I just failed to do that with my first reading.

The example is fine.